docat/app.py

```diff
@@ -38,10 +37,18 @@ def upload(project, version):
     base_path = project_base_path / version
     target_file = base_path / secure_filename(uploaded_file.filename)
 
+    if base_path.is_dir():
```

Comment:
So symlinks (tags) can still be overwritten by new versions?
Is this intended behaviour?

Reply:
good question, you are right currently this implementation allows to overwrite a tag with a actual thing maybe this should be verboten

Reply:
i will fix that

Reply:
turns out `is_dir` is true for a symlink which point to a dir but i will change it to `exists` regardless

Reply:
and i added a test ;)